src/Azure/Provider.php

```diff
@@ -91,17 +98,45 @@ protected function mapUserToObject(array $user)
      */
     protected function getTokenFields($code)
```

Comment:
`getTokenFields ` seems just like a copy of `\Laravel\Socialite\Two\AbstractProvider::getCodeFields` so I guess this can be removed

Reply:
if you mean ```\Laravel\Socialite\Two\AbstractProvider::getTokenFields``` instead of `\Laravel\Socialite\Two\AbstractProvider::getCodeFields`  --> ok removed `getTokenFields` from child class